src/System.ComponentModel.Composition/tests/Configuration.props

```diff
@@ -1,13 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why are you deleting the tests Configurations.props? 

This one would be like:

```xml
<BuildConfigurations>
  netcoreapp;
  uap;
</BuildConfigurations>

Reply:
I meant to rename to Configurations.props (plural). forgot to add the new file

thanks for noticing